INSTALL.md

```diff
@@ -491,6 +491,14 @@ DeePMD-kit - Deep Potential Molecular Dynamics. Support for DeePMD-kit can be en
 - Add `-D__SMEAGOL` to DFLAGS, `-I$(LIBSMEAGOL_DIR)/obj` to FCFLAGS and
   `-L$(LIBSMEAGOL_DIR)/lib -lsmeagol` to LIBS
 
+### 2z. TREXIO (optional, unified computational chemistry format)
```

Comment:
The section is numbered 2z. even though it exists already, but we have run out of letters...